src/components/common/Balance.test.js

```diff
@@ -14,6 +14,7 @@ describe('<Balance.js>', ()=>{
     it('balance should return properly for non 0 for 0.0987',()=>{
         let wrapper = shallow(<Balance amount={contextNull} />)
         expect(wrapper.contains("<0.00001")).toEqual(true)
+                        
```

Comment:
nit: not clear why empty line added here

Reply:
done